utils/memtester/Makefile

```
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2007-2023 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=memtester
+PKG_VERSION:=4.6.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://pyropus.ca/software/memtester/old-versions/
+PKG_HASH:=c9fe4eb7e80c8cef5202f9065c4c0682f5616647c0455e916a5700f98e3dbb2e
+PKG_MAINTAINER:=GuoWei Lim <limguowei@gmail.com>
```

```suggestion

PKG_LICENSE:=GPL-2.0-only
PKG_LICENSE_FILES:=COPYING
PKG_MAINTAINER:=Aleksaner Jan Bajkowski <olek2@wp.pl>
```

add copyright info and assign yourself as maintainer, as the original author is no longer active